docs/effects/sounds.md

```diff
@@ -1,78 +1,63 @@
 Sounds
 ======
 
-!!!Note
-    This guide was written at a time when the latest Forge build was build 1907 for Minecraft 1.9.0. It should be valid for any Forge version that has the new Forge registry system.
-
-Goals
------
-
-  * Describe a simple way to implement sounds in Forge
-  * There are many, many versions of `playSound` in Minecraft. Describe what they all do
-
 Terminology
 -----------
 
 | Term | Description |
-|-|-|
-| Sound Events | Something that triggers a sound effect. Examples include `"minecraft:block.anvil.hit"` or `"botania:spreaderFire"`. |
+|----------------|----------------|
+|  Sound Events  | Something that triggers a sound effect. Examples include `"minecraft:block.anvil.hit"` or `"botania:spreaderFire"`. |
 | Sound Category | The category of the sound, for example `"player"`, `"block"` or simply `"master"`. The sliders in the sound settings GUI represent these categories. |
-| Sound File | The literal file on disk that is played, usually an .ogg file. |
+|   Sound File   | The literal file on disk that is played, usually an .ogg file. |
 
-Sounds.json
------------
+`sounds.json`
+-------------
 
-This json should be located in your base asset directory:
-```
-src/main/resources/assets/[mod id]/sounds.json
-```
-This indicates to the vanilla resource system what Sound Events you declare and what Sound Files those events use.
+This json should be located in the `assets` directory of a mod: `src/main/resources/assets/<mod id>/sounds.json`. This file declares sound events and identifies which sound files they use.
```

Comment:
Personally, I think path would be better using `<modID>` so there's no space in it. Just an opinion though.